frappe/website/page_renderers/template_page.py

```diff
@@ -212,19 +212,13 @@ def set_properties_via_comments(self):
 
 	def run_pymodule_method(self, method_name):
 		if hasattr(self.pymodule, method_name):
-			try:
-				import inspect
-
-				method = getattr(self.pymodule, method_name)
-				if inspect.getfullargspec(method).args:
-					return method(self.context)
-				else:
-					return method()
-			except (frappe.PermissionError, frappe.DoesNotExistError, frappe.Redirect):
-				raise
-			except Exception:
-				if not frappe.flags.in_migrate:
-					frappe.errprint(frappe.utils.get_traceback())
```

Comment:
We used to silently log all the exceptions... now we'll be responding with an error page. Not yet sure that this will not break anything 👀

But, the change seems correct to me.